src/inspector/index.css

```diff
@@ -2,53 +2,89 @@
 | Copyright (c) Jupyter Development Team.
 | Distributed under the terms of the Modified BSD License.
 |----------------------------------------------------------------------------*/
-.jp-Inspector .p-TabBar {
-  min-width: 300px;
-  min-height: 24px;
-  color: var(--jp-ui-font-color1);
-  font-size: 13px;
+
+/*-----------------------------------------------------------------------------
+| Variables
+|----------------------------------------------------------------------------*/
+
+:root {
+  --jp-private-inspector-tab-height: 24px;
+  --jp-private-inspector-tab-width: 60px;
+
 }
 
+/*-----------------------------------------------------------------------------
+| Inspector
+|----------------------------------------------------------------------------*/
 
-.jp-Inspector .p-TabBar-content {
-  min-width: 0;
-  padding-right: 1px;
+.jp-Inspector .p-TabPanel-stackedPanel {
+  border: none;
 }
 
 
-.jp-Inspector .p-TabBar-tab {
-  flex-basis: 144px;
-  min-height: 23px;
-  min-width: 35px;
-  padding: 0px 7px;
-  background: #EEEEEE;
-  border: var(--jp-border-width) solid var(--jp-border-color1);
-  transform: translateY(-1px);
-  border-bottom: none;
-  margin-right: -1px;
+/* default inspector items: hints */
+.jp-InspectorItem.jp-HintsInspectorItem,
+.jp-InspectorItem.jp-HintsInspectorItem .p-Widget {
+  background: var(--jp-layout-color1);
 }
 
 
-.jp-Inspector .p-TabBar-tab.p-mod-current {
-  min-height: 24px;
-  background: #FFFFFF;
-  border-top: var(--jp-border-width) solid var(--jp-brand-color1);
+.jp-InspectorItem.jp-HintsInspectorItem .jp-Toolbar {
+  display: none;
 }
 
 
-.jp-Inspector .p-TabBar-tab:hover:not(.p-mod-current) {
-  background: #FAFAFA;
+/* default inspector items: details */
+.jp-InspectorItem.jp-DetailsInspectorItem .p-Widget {
+  background: var(--jp-layout-color1);
 }
 
 
-.jp-Inspector .p-TabBar-tabIcon,
-.jp-Inspector .p-TabBar-tabText {
-  line-height: 24px;
+/*-----------------------------------------------------------------------------
+| Inspector tabs
+|----------------------------------------------------------------------------*/
+
+.jp-Inspector .p-TabBar {
+  min-width: 300px;
+  min-height: var(--jp-private-inspector-tab-height);
+  color: var(--jp-ui-font-color1);
+  font-size: var(--jp-ui-font-size1);
 }
 
 
+.jp-Inspector .p-TabBar .p-TabBar-tab {
```

Comment:
This is now merged, but the cheaper way to increase specificity would be to do something like `.jp-Inspector.p-Widget .p-TabBar-tab`. Descendant selectors are the most expensive for the browser to match, so should be used as little as possible.